src/mlpack/methods/ann/layer/glimpse_impl.hpp

```diff
@@ -224,6 +224,9 @@ void Glimpse<InputDataType, OutputDataType>::serialize(
   ar & BOOST_SERIALIZATION_NVP(depth);
   ar & BOOST_SERIALIZATION_NVP(scale);
   ar & BOOST_SERIALIZATION_NVP(inputWidth);
+  ar & BOOST_SERIALIZATION_NVP(inputHeight);
```

Comment:
Hm, looks like this was part of https://github.com/mlpack/mlpack/pull/1905, but I think there is no need to remove this one from here.